src/main/java/seedu/partyplanet/ui/MainWindow.java

```diff
@@ -196,11 +209,33 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
+            if (commandResult.isToggleTheme()) {
+                setTheme(commandResult.getTheme());
+            }
+
             return commandResult;
         } catch (CommandException | ParseException e) {
             logger.info("Invalid command: " + commandText);
             resultDisplay.setFeedbackToUser(e.getMessage());
             throw e;
         }
     }
+
+    /**
+     * Set themes dynamically based on user command.
+     */
+    private void setTheme(Theme theme) {
+        scene.getStylesheets().clear();
+        scene.getStylesheets().addAll(Theme.getStyleSheets(theme));
+        logic.setGuiSettings(new GuiSettings(theme));
+        this.theme = theme;
+    }
+    @FXML
+    public void setThemePastel() {
```

Comment:
Optional: can be private?
```suggestion
    private void setThemePastel() {
```